marklogic-client-api/src/test/java/com/marklogic/client/test/dataservices/MarklogicSourceConnector.java

```diff
@@ -0,0 +1,62 @@
+package com.marklogic.client.test.dataservices;
+
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.connect.connector.Task;
+import org.apache.kafka.connect.source.SourceConnector;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class MarklogicSourceConnector extends SourceConnector {
+    private String filename;
+    String FILE_CONFIG = "file_config";
+    String TOPIC_CONFIG = "topic_config";
+    private Map<String, String> config;
+    private String topic;
+    @Override
+    public void start(Map<String, String> props) {
+       // filename = props.get(FILE_CONFIG);
+        //topic = props.get(TOPIC_CONFIG);
+        config = props;
+
+    }
+
+
+    @Override
+    public Class<? extends Task> taskClass() {
+
+        return MarklogicSourceTask.class;
+    }
+
+    @Override
+    public List<Map<String, String>> taskConfigs(int maxTasks) {
+// number of tasks = number of forests
+        // forestname passed to tasks and tasks puts it in workunit
+        // map - "forestname", forest_1 etc
+
+
+        final List<Map<String, String>> configs = new ArrayList<>(3);
+        for (int i = 0; i < 3; i++) {
+            config.put("forestName","java-unittest-"+(i+1));
```

Comment:
Shouldn't each item in the list be a different Map?  If it's the same Map, won't the last put() to "forestName" be the assignment for all tasks?  

Would it make sense to create a new Map within the loop, copy the properties with putAll(), and then set the "forestName"?